examples/parallel/distributed_backend_simple.py

```diff
@@ -0,0 +1,57 @@
+"""
+Using distributed for single_machine parallel computing
+========================================================
+
+This example shows the simplest usage of the dask `distributed
+<https://distributed.readthedocs.io>`__ backend, on the local computer.
+
+This is useful for prototyping a solution, to later be run on a truly
+distributed cluster, as the only change to be made is the address of the
+scheduler.
+
+Another realistic usage scenario: combining dask code with joblib code,
+for instance using dask for preprocessing data, and scikit-learn for
+machine learning. In such a setting, it may be interesting to use
+distributed as a backend scheduler for both dask and joblib, to
+orchestrate well the computation.
+
+"""
+
+###############################################################################
+# Setup the distributed client
+###############################################################################
+from distributed import Client
+# Typically, to execute on a remote machine, the address of the scheduler
+# would go there
+client = Client()
+
+# Recover the address
+address = client.scheduler_info()['address']
+
+# This import registers the dask backend for joblib
+import distributed.joblib  # noqa
+
+###############################################################################
+# Run parallel computation using dask.distributed
+###############################################################################
+
+import time
+import joblib
+
+
+def long_running_function(i):
+    time.sleep(.1)
+    return i
+
+
+with joblib.parallel_backend('dask.distributed', scheduler_host=address):
```

Comment:
I would even add `backend='dask.distributed'` or add a small discussion after the title (it seems a bit empty there).

Reply:
But I am not 100% sure.

Reply:
This should suffice if you have already created a client (requires master).  You also don't need the `address = ` line above

```python
with joblib.parallel_backend('dask'):
```

Reply:
Now backend='dask' is very confusing, because, as @TomAugspurger was explaining to me, by default dask doesn't use the distributed backend, but a threading one.

I'll wait for a release of distributed to update this example, as I would like it to run with released version.

Reply:
Right, but you'll never use this with the legacy threaded scheduler, it isn't sufficiently flexible to handle Joblib's dynamism.  There is only one relevant use case for Dask here, and it's the newer scheduler.